CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+Contributing to EssentialsX
+===========================
+
+Want to help improve EssentialsX? There are numerous ways you can contribute to the project.
+
+If you'd like to make a financial contribution to the project, you can join our [Patreon](https://www.patreon.com/essentialsx/),
+or to make a one-off donation you can visit our [Ko-fi page](https://ko-fi.com/essentialsx). If you can't make a
+donation, don't worry! There's lots of other ways to contribute:
```

Comment:
Seems redundant
```suggestion
donation, don't worry!
```

Reply:
I prefer this as it introduces the next section. (also this text isn't new, it's been in README for a while)

Reply:
Yeah this was minor. Just thought it could be cleaned up more since just above that, this section was introduced as a way of contributing.